### PR TITLE
Fix S2934 FN: Support unsigned right shift operator (>>>)

### LINKED ISSUE #6296: Fix S2934 FN: Support unsigned right shift operator (>>>)
### Description

FN for S2934 (GenericReadonlyFieldPropertyAssignment).

Using C#11 unsigned right shift operator (>>>), the rule is not raising an issue.

### Repro steps

```csharp
 interface IPoint
    {
        int X { get; set; }
        int Y { get; set; }
    }

    class PointManager<T> where T : IPoint
    {
        readonly T point;  // this could be a struct

        public void MovePointVertically(int newX)
        {
            point.X >>>= 4; // FN
        }
    }

```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.

### PR DESCRIPTION
**Pull request by gregory-paidis-sonarsource**
Fixes #6296

### CODE REVIEW

File: analyzers/src/SonarAnalyzer.CFG/ShimLayer/SyntaxKindEx.cs [hunk 1]
```diff
@@ -9,6 +9,8 @@ public static class SyntaxKindEx
     {
         public const SyntaxKind DotDotToken = (SyntaxKind)8222;
         public const SyntaxKind QuestionQuestionEqualsToken = (SyntaxKind)8284;
+        public const SyntaxKind GreaterThanGreaterThanGreaterThanToken = (SyntaxKind)8286;
```
Comment (cristian-ambrosini-sonarsource) [inline analyzers/src/SonarAnalyzer.CFG/ShimLayer/SyntaxKindEx.cs:12]: This is not used, maybe we can remove it to add less code as possible

**Review by cristian-ambrosini-sonarsource — CHANGES_REQUESTED** [review]
LGTM! I suggest to remove 1 line

**Review by cristian-ambrosini-sonarsource — APPROVED** [review]

File: analyzers/src/SonarAnalyzer.CSharp/Rules/GenericReadonlyFieldPropertyAssignment.cs [hunk 2]
```diff
@@ -59,7 +59,8 @@ protected override void Initialize(SonarAnalysisContext context)
                 SyntaxKind.OrAssignmentExpression,
                 SyntaxKind.LeftShiftAssignmentExpression,
                 SyntaxKind.RightShiftAssignmentExpression,
-                SyntaxKindEx.CoalesceAssignmentExpression);
+                SyntaxKindEx.CoalesceAssignmentExpression,
+                SyntaxKindEx.GreaterThanGreaterThanGreaterThanEqualsToken);
```
Comment (csaba-sagi-sonarsource) [inline analyzers/src/SonarAnalyzer.CSharp/Rules/GenericReadonlyFieldPropertyAssignment.cs:63]: Use. At this point, registering to the token or the UnsignedRightShiftAssignmentExpression might give back the same syntax node but that might change in the future. This is the correct way to do it. (In a weird case that token can appear in other constructs too.)
```suggestion
                SyntaxKindEx.UnsignedRightShiftAssignmentExpression);
```

**Review by csaba-sagi-sonarsource — CHANGES_REQUESTED** [review]
Left a couple of comments.

**Review by csaba-sagi-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**